riva/clients/nmt/streaming_s2t_client.cc

```diff
@@ -256,6 +256,8 @@ StreamingS2TClient::PostProcessResults(std::shared_ptr<S2TClientCall> call, bool
     VLOG(1) << "Latency:" << lat << std::endl;
     latencies_.push_back(lat);
   }
+  std::ofstream result_file(nmt_text_file_, std::ios::app);
```

Comment:
can we open/close file in the constructor/destructor? like how ASR client is doing. We can put the std::ofstream object as a class member variable.